torchtext/experimental/vectors.py

```diff
@@ -120,14 +123,16 @@ def GloVe(name="840B", unk_tensor=None, root='.data', validate_file=True):
                             "����������������������������������������������������������������������"
                             "������������������������������������������������������", 140649)
     urls = {
-        '42B': 'https://nlp.stanford.edu/data/glove.42B.300d.zip',
-        '840B': 'https://nlp.stanford.edu/data/glove.840B.300d.zip',
-        'twitter.27B': 'https://nlp.stanford.edu/data/glove.twitter.27B.zip',
-        '6B': 'https://nlp.stanford.edu/data/glove.6B.zip',
+        "42B": "https://nlp.stanford.edu/data/glove.42B.300d.zip",
+        "840B": "https://nlp.stanford.edu/data/glove.840B.300d.zip",
+        "twitter.27B": "https://nlp.stanford.edu/data/glove.twitter.27B.zip",
+        "6B": "https://nlp.stanford.edu/data/glove.6B.zip",
     }
 
     url = urls[name]
-    cached_vectors_file_path = os.path.join(root, os.path.splitext(os.path.basename(url))[0] + '.pt')
+    file_name = "glove.{}.{}d.txt".format(name, str(dim))
```

Comment:
Should we check if dim is support for the specific name?

Reply:
Do you think I should have a dict like this? And then check that the file name exists within the dict? https://github.com/pytorch/text/blob/fd764ad02968e6310d28ee3c496e1d1d1157aa3e/torchtext/vocab.py#L523